plugins/calibrecompanion.koplugin/main.lua

```diff
@@ -80,6 +81,17 @@ function CalibreCompanion:find_calibre_server()
     end
 end
 
+function CalibreCompanion:checkCalibreServer(host, port)
+    local socket = require("socket")
+    local client, err = socket.connect(host, port)
+    DEBUG(client, err)
```

Comment:
Might be better to convert the whole thing to logger.dbg rather than adding new DEBUG in here?

Also as currently written I imagine it'll default to info as opposed to dbg.